pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -219,9 +197,6 @@ func (state PipelineRunState) GetRunsStatus(pr *v1beta1.PipelineRun) map[string]
 		if !rprt.IsCustomTask() {
 			continue
 		}
-		if rprt.Run == nil && rprt.ResolvedConditionChecks == nil {
```

Comment:
why is this nil check going away but the one above (for taskRuns) sticking around?

Reply:
In practice, `rprt.ResolvedConditionChecks` for a `Run` was always non-nil, so this `if` never really did anything in the first place, if I'm remembering correctly.

Reply:
More concretely, I tried to ensure that the tests had as little changed as possible, and getting rid of this made no difference to the test results, so...

Reply:
this `if` was never reached...